actpool/actpool_test.go

```diff
@@ -9,6 +9,7 @@ package actpool
 import (
 	"bytes"
 	"context"
+	"github.com/iotexproject/iotex-address/address"
```

Comment:
move to line 23 below

Reply:
also move line 18-21 together with other `/iotex-core`

Reply:
done..imports sorted as per go-fmt